### PR TITLE
Fy 09 UI fixes

### LINKED ISSUE #17: Fix: Minor Issues and UI Fixes
Fix Some UI and Minor issues

### PR DESCRIPTION
**Pull request by sadathshariff**

### CODE REVIEW

File: src/components/Sidenav/Sidenav.css [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
-  flex-basis: 20%;
   border-radius: 5px;
-  height: 100vh;
-  box-shadow: 0 6px 6px 2px rgba(255, 255, 255, 0.5);
+  height: 100%;
+  box-shadow: 0 6px 6px 4px rgb(0 0 0 / 50%);
+  display: flex;
+  flex-direction: column;
+  width: 15rem;
+  height: 100%;
```
Comment (nishasen) [inline src/components/Sidenav/Sidenav.css]: you have given height 2 times

File: src/pages/VideoPage/VideoPage.jsx [hunk 2]
```diff
@@ -51,17 +51,17 @@ export const VideoPage = () => {
...
   return (
     <>
       <div className="videoplayer-container">
         <YouTube
           class="video-iframe"
           videoId={videoId}
           opts={opts}
-          onReady={handlePlay}
+          onPlay={() => {
+            // !history?.map((item) => item._id === data._id) &&
```
Comment (nishasen) [inline src/pages/VideoPage/VideoPage.jsx]: remove commented code

**Review by nishasen — APPROVED** [review]
Everything looks good, suggested small changes

File: src/components/Sidenav/Sidenav.css (hunk 1 above)
Comment (sadathshariff) [reply src/components/Sidenav/Sidenav.css]: Thanks For Pointing it out,Nisha

**Review by vaibhav-mahalle — APPROVED** [review]
just a minor suggestion keep the commit messages consistent. everything else looks good,
Look into the mistakes pointed out by nisha.

**Pull request merged**